roles/semanage_add_verify/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Check for presence of container_file_t type
+  command: seinfo --type=container_file_t
+  register: seinfo
+  ignore_errors: True
+
+- name: Set SELinux type to be used
+  set_fact:
+    selinux_type: '{{ "container_file_t" if seinfo.rc == 0 else "svirt_sandbox_file_t" }}'
+    dir_match: '/var/lib/kubelet(/.*)?'
```

Comment:
I would like to move these types of global `set_fact` tasks into one role we can run at the beginning of each playbook.

Not for this review, but something we should do in the near future.